crates/epaint/src/shape.rs

```diff
@@ -288,6 +288,19 @@ impl Shape {
             .into()
     }
 
+    /// The entire [`Galley`] will be rendered with the given opacity in gamma space.
+    #[inline]
+    pub fn galley_with_opacity_factor(
```

Comment:
Same here - no need for this Imho. Users can just use the `TextShape::new(…).with_opacity_factor(…)`

Reply:
Applied in latest commit